app/src/main/java/se/inera/intyg/minaintyg/certificate/CertificateController.java

```diff
@@ -69,4 +79,35 @@ public void sendCertificateToRecipient(@PathVariable String certificateId) {
             .build()
     );
   }
+
+  @PostMapping("/{certificateId}/print")
+  public ResponseEntity printCertificate(
```

Comment:
Verify if we need to add "produces"-annotation for pdf.

Reply:
From what I found out PostMapping is handled the same as RequestMapping so I added the produces option according to that